help/gory_details/_posts/2021-04-26-author-search.md

```diff
@@ -0,0 +1,103 @@
+
+So what started as few characters (one query) is turned behind the scenes into a massive and very complicated query. There is a constant battle between the desire to be precise, and the desire to include everybody who should be there (recall). Yet all of this without really knowing much about the person, all we have is just a bunch of characters. The logic is complex, it takes advantage of what ADS knows about the authors and includes a knowledge base (plus business logic) but it is far from being able to understand 'intentions' of the user; and it has no real model of a person (and their relationships).
+
+If it produces unexpected results, it usually is because of these reasons:
+
+- there is a bug somewhere in this insanely complex chain -- a bug that was not caught by even the more insanely large suite of unittests (the frequency of this being the case is diminishing rapidly with time, and it happens rarely - but it is really difficult to say with certainty until the whole testsuite is examined; so it should alway be the first option)
+- the curated knowledge base is too broad or too narrow: i.e. it is missing a connection, or has connection too many -- and joins unrelated names together
+- haphazard interferences: wrong data indexed, wrong or outdated versions of synonyms/transliterations deployed, parts of the production cluster being down, parts of the backoffice processing pipeline being down....
+
+
+### Intermezzo #1
+
+Parts of the author search logic can be turned on/off; here are some of the most useful options:
+
+- `=author:"adamzuk, p"` the equal sign will instruct our parser not to look for transliterations/synonyms/variants
+- `author:/adamzuk,\ p/` -- you can use regular expressions, basically throwing away all the fancy logic; but mind that in the real world people really have many different names; so with this approach you will only find papers that contain just the characters of your pattern
+- `author:"adamczuk, p"^0.7` -- the funny tilda float number activates a fuzzy search for the similarly sounding names. You'll need to play with the modifier until you get names that work for a particular use case (the range is 0.1-1.0)
```

Comment:
"tilda" -> "tilde" but also, you're not using the `~` you're using the `^` here. Also, link to the help doc where you now explain fuzzy searching